media-gfx/synnefo/synnefo-9999.ebuild

```diff
@@ -0,0 +1,43 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=5
+
+inherit cmake-utils
+
+DESCRIPTION="Qt front end for the Oyranos Color Management System"
+HOMEPAGE="https://github.com/oyranos-cms/Synnefo"
+if [[ ${PV} == "9999" ]] ; then
+	EGIT_REPO_URI="https://github.com/oyranos-cms/Synnefo.git"
+	inherit git-r3
+	KEYWORDS=""
+#else
+#	SRC_URI="https://github.com/oyranos-cms/Synnefo/archive/${PV}.tar.gz -> ${P}.tar.gz"
+#	KEYWORDS="~amd64 ~x86"
+fi
+
+LICENSE="BSD-2"
+SLOT="0"
+IUSE="qt4"
```

Comment:
This is an interesting problem. While we (Qt team) have yet to agree on a formal policy, we are leaning towards reversing this logic and using a qt5 USE flag instead.